test/AssetPool.test.js

```diff
@@ -227,6 +227,29 @@ describe("AssetPool", () => {
         ).to.be.closeTo("45454545454545454545", assertionPrecision)
       })
     })
+
+    context(
+      "when collateral tokens sent to asset pool outside of cov pools",
```

Comment:
I am not sure if the name of this test explains well what we are trying to check. It is fine if someone does a plain transfer, it's how ETH->KEEP transition from purchased bonds will work.

Reply:
@pdyraga 
I think it mainly affects how many `underwriter tokens` will be minted for the `underwriter` in exchange for the `collateral tokens` they are depositing. 
Any suggestion how I could name this test?
Maybe `when collateral tokens in asset pool are increased via plain transfer`?

Reply:
Correct. What I mean is that:
- "when collateral tokens sent to asset pool outside of cov pools"
- "when collateral tokens in asset pool are increased via plain transfer"

are both perfectly valid cases and if we say that the deposit should always revert in such situations, this is wrong because it is not what is actually expected. We do not expect `deposit` to always revert if ETH->KEEP swap happened from purchased signer bonds and that KEEP funded `AssetPool`. Similarly, we do not expect `deposit` to always revert if `RewardsPool` released KEEP tokens and funded `AssetPool`.

We do however expect `deposit` to revert if the amount of COV minted would be 0.

So I suggest we name this scenario:
```
  context("when minting 0 underwriter tokens", () => {
    // (...)
    it("should revert", async () => {
}
```

because this is what we are actually preventing from and asserting.

Reply:
OK, now I get it.
Fixed in https://github.com/keep-network/coverage-pools/pull/145/commits/61952dd033344000712d1a74a06244b2f38331dc.